### DISCUSSION TITLE
About TI simulation

### DISCUSSION THREAD

**Discussion by lingoodluck** [simnibs/simnibs#412] [8. Simulation workflows]
I have a question. The process set directly through the GUI should be the same as the actual experimental operation on the human body. The TI results I obtained directly using the GUI are different from the TI results achieved through the code you provided. In the code, you calculated TImax using a formula, but in the actual experiment, can I achieve the same effect by applying direct current (DC) electricity? I feel that the actual operation will be similar to the results from the GUI. In the actual operation, can I use direct current to achieve the same field distribution as TI?

**Comment by diedrim** [comment]
It's not possible to set up TI in the GUI, only normal TDCS simulation. You have to simulate two TDCS setups and combine their results in postprocessing step to get TI results. Therefore TI simulation in the GUI is not possible.

**Reply by lingoodluck** [reply]
Hello! When I was trying the TI simulation, I placed two pairs of electrodes at F5-P5 and F6-P6 respectively. I looked at the simulation results of one pair of tDCS and found that the electric field at the compact bone suddenly dropped significantly in a triangular area on the other side. Can someone help explain this? Figure 1 shows the simulation results of one pair of tDCS, and Figure 2 shows the results of the TI simulation.
![图片3](https://github.com/user-attachments/assets/6f1c0796-47ec-48f6-ba1e-93213a2edcd2)
![图片2](https://github.com/user-attachments/assets/e5444e4a-8b0c-45b6-b5ff-302e10096144)

**Reply by TorgeW** [reply]
https://github.com/simnibs/simnibs/discussions/474
